tutorials/promscale-benefits.md

```diff
@@ -0,0 +1,97 @@
+## Why use Promscale & TimescaleDB to store Prometheus metrics [](why-promscale)
```

Comment:
Need to raise the hierarchy of all headlines (## -> #, etc.).  I assume this is b/c they were previously all part of one page?